src/data/staticData/extensionData.ts

```diff
@@ -36,7 +36,7 @@ export interface ExtensionData {
   neededColonyPermissions: ColonyRole[];
   initializationParams?: ExtensionInitParams[];
   uninstallable: boolean;
-  enabledExtensionBody?: () => ReactNode;
+  enabledExtensionBody?: (props: WhitelistProps) => ReactNode;
```

Comment:
I would suggest to do it this way not to depend on any component props since it will the only one for all the extensions
```
  enabledExtensionBody?: (colonyAddress: Address) => ReactNode;
```

Reply:
I'm gonna make an `ExtensionBodyProps` interface then, where all the common props, like `colonyAddress`, will be added